config/params/testnet_holesky_config.go

```diff
@@ -26,6 +26,7 @@ func HoleskyConfig() *BeaconChainConfig {
 	cfg.MinGenesisTime = 1695902100
 	cfg.GenesisDelay = 300
 	cfg.ConfigName = HoleskyName
+	cfg.GenesisValidatorsRoot = [32]byte{}
```

Comment:
these values are wrong, you are setting all empty byte arrays... we wouldn't need hardcoded values if it's all empty byte arrays